enclave-manager/web/src/emui/catalog/storage.ts

```diff
@@ -0,0 +1,26 @@
+import { KurtosisPackage } from "../../client/packageIndexer/api/kurtosis_package_indexer_pb";
+import { isDefined, stringifyError } from "../../utils";
+
+const SAVED_PACKAGES_LOCAL_STORAGE_KEY = "kurtosis-saved-packages";
+
+export const storeSavedPackages = (kurtosisPackages: KurtosisPackage[]) => {
+  localStorage.setItem(
+    SAVED_PACKAGES_LOCAL_STORAGE_KEY,
+    JSON.stringify(kurtosisPackages.map((kurtosisPackage) => kurtosisPackage.name)),
+  );
+};
```

Comment:
@Dartoxian what do you think about adding version number to this storage object. Like `"version":1` or something? Just so that we can later on migrate people if we make a breaking change to this data?